src/main/resources/assets/js/app/view/ContentItemPreviewPanel.ts

```diff
@@ -300,7 +300,14 @@ export class ContentItemPreviewPanel
             if (allows) {
                 this.setPreviewType(PREVIEW_TYPE.MEDIA);
                 if (this.isVisible()) {
-                    this.frame.setSrc(api.util.UriHelper.getRestUri(`content/media/${contentSummary.getId()}?download=false#view=fit`));
+                    const uri = api.util.UriHelper.getRestUri(`content/media/${contentSummary.getId()}?download=false#view=fit`);
+
+                    if(this.frame.getSrc() !== uri) {
+                        this.frame.setSrc(uri);
+                    } else {
+                        this.frame.refresh();
+                    }
+
```

Comment:
I suggest moving this logic (`if...else...`) inside `setSrc` method of the `IFrame` class, because we need this kind of refresh to happen not only in `ContentItemPreviewPanel` but in general for any iframe, whenever URL has changed. Which means there will be no changes here, but only in `api.dom.IFrameEl`